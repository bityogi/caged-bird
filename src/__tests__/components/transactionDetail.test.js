import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'

import TransactionDetail from 'components/transactionDetail';
import store from 'store';

describe('Landing', () => {
  it('renders correctly', () => {

    const detail = {
      loading: false,
      data: {
        id: 1234,
        client: 'Conger Inv',
        coin: 'Bitcoin',
        account: 'Offshore',
        coldWallet: 'Wallet 1',
        hotWallet: '1f4567yggt',
        amount: 231.66,
        fee: 12.55,
        memo: 'This transaction is good and will be executed soon',
        genTime: '2018-03-07T09:18:26-05:00'
      }
    }
    const rendered = renderer.create(
      <Provider store={store}>
        <MuiThemeProvider>
          <TransactionDetail detail={detail}/>
        </MuiThemeProvider>
      </Provider>

    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });

});
