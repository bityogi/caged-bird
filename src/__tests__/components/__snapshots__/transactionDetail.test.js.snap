// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders correctly 1`] = `
<div
  style={
    Object {
      "muiPrepared": true,
      "padding": "8px 0px 8px 0px",
    }
  }
>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <svg
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fill": "#00838f",
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 0,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Transaction ID
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 0,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            person_pin
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Client
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 0,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            album
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Coin
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 12,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            account_balance
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Account
          </div>
          <div
            style={
              Object {
                "WebkitBoxOrient": null,
                "WebkitLineClamp": null,
                "color": "rgba(0, 0, 0, 0.54)",
                "display": null,
                "fontSize": 14,
                "height": 16,
                "lineHeight": "16px",
                "margin": 0,
                "marginTop": 4,
                "muiPrepared": true,
                "overflow": "hidden",
                "textOverflow": "ellipsis",
                "whiteSpace": "nowrap",
              }
            }
          >
            Offshore
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 0,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            account_balance_wallet
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Cold Wallet
          </div>
          <p
            style={
              Object {
                "WebkitBoxOrient": "vertical",
                "WebkitLineClamp": 2,
                "color": "rgba(0, 0, 0, 0.54)",
                "display": "-webkit-box",
                "fontSize": 14,
                "height": 36,
                "lineHeight": "18px",
                "margin": 0,
                "marginTop": 4,
                "muiPrepared": true,
                "overflow": "hidden",
                "textOverflow": "ellipsis",
                "whiteSpace": null,
              }
            }
          >
            <br />
            undefined
          </p>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 0,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            camera_enhance
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Hot Wallet
          </div>
          <p
            style={
              Object {
                "WebkitBoxOrient": "vertical",
                "WebkitLineClamp": 2,
                "color": "rgba(0, 0, 0, 0.54)",
                "display": "-webkit-box",
                "fontSize": 14,
                "height": 36,
                "lineHeight": "18px",
                "margin": 0,
                "marginTop": 4,
                "muiPrepared": true,
                "overflow": "hidden",
                "textOverflow": "ellipsis",
                "whiteSpace": null,
              }
            }
          >
            <br />
            undefined
          </p>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 12,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            monetization_on
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Amount
          </div>
          <div
            style={
              Object {
                "WebkitBoxOrient": null,
                "WebkitLineClamp": null,
                "color": "rgba(0, 0, 0, 0.54)",
                "display": null,
                "fontSize": 14,
                "height": 16,
                "lineHeight": "16px",
                "margin": 0,
                "marginTop": 4,
                "muiPrepared": true,
                "overflow": "hidden",
                "textOverflow": "ellipsis",
                "whiteSpace": "nowrap",
              }
            }
          >
            231.66
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 12,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            attach_money
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Fee
          </div>
          <div
            style={
              Object {
                "WebkitBoxOrient": null,
                "WebkitLineClamp": null,
                "color": "rgba(0, 0, 0, 0.54)",
                "display": null,
                "fontSize": 14,
                "height": 16,
                "lineHeight": "16px",
                "margin": 0,
                "marginTop": 4,
                "muiPrepared": true,
                "overflow": "hidden",
                "textOverflow": "ellipsis",
                "whiteSpace": "nowrap",
              }
            }
          >
            12.55
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 12,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            description
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Memo
          </div>
          <div
            style={
              Object {
                "WebkitBoxOrient": null,
                "WebkitLineClamp": null,
                "color": "rgba(0, 0, 0, 0.54)",
                "display": null,
                "fontSize": 14,
                "height": 16,
                "lineHeight": "16px",
                "margin": 0,
                "marginTop": 4,
                "muiPrepared": true,
                "overflow": "hidden",
                "textOverflow": "ellipsis",
                "whiteSpace": "nowrap",
              }
            }
          >
            This transaction is good and will be executed soon
          </div>
        </div>
      </div>
    </span>
  </div>
  <div>
    <span
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      open={null}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "background": "none",
          "backgroundColor": null,
          "border": 10,
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 16,
          "fontWeight": "inherit",
          "lineHeight": "16px",
          "margin": 0,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "textDecoration": "none",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "verticalAlign": null,
        }
      }
      tabIndex={0}
    >
      <div
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
      >
        <div
          style={
            Object {
              "marginLeft": 0,
              "muiPrepared": true,
              "paddingBottom": 2,
              "paddingLeft": 50,
              "paddingRight": 16,
              "paddingTop": 10,
              "position": "relative",
            }
          }
        >
          <span
            className="material-icons"
            color="#757575"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "#00838f",
                "display": "block",
                "fontSize": 24,
                "height": 24,
                "left": 4,
                "margin": 12,
                "muiPrepared": true,
                "position": "absolute",
                "top": 0,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
          >
            access_time
          </span>
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            Gen Time
          </div>
        </div>
      </div>
    </span>
  </div>
</div>
`;
